cms/utils/check.py

```diff
@@ -354,14 +354,15 @@ def check_placeholder_fields(output):
 
     with output.section("PlaceholderField") as section:
         for model, model_admin in site._registry.items():
-            ph_fields = [field for field in model._meta.get_fields() if isinstance(field, PlaceholderField)]
-            if len(ph_fields) == 0:
-                continue
-
-            if not isinstance(model_admin, PlaceholderAdminMixin):
-                section.error(
-                    "%s does not subclass of PlaceholderAdminMixin" % model_admin
-                )
+            if getattr(model._meta, 'get_fields', None):
+                ph_fields = [field for field in model._meta.get_fields() if isinstance(field, PlaceholderField)]
+                if len(ph_fields) == 0:
+                    continue
+
+                if not isinstance(model_admin, PlaceholderAdminMixin):
+                    section.error(
+                        "%s does not subclass of PlaceholderAdminMixin" % model_admin
+                    )
```

Comment:
@ton77v This looks good to me! 

Only thought I have is: Can you **add an `else` clause** that issues a warning (`section.warn("...")`) that the model `model` was not checked if it contained placeholders. In general that should be ok - just to not miss an unintended inheritance.